cmd/Makefile.am

```diff
@@ -273,6 +273,18 @@ snap_confine_snap_confine_extra_libs += $(APPARMOR_LIBS)
 endif  # STATIC_LIBAPPARMOR
 endif  # APPARMOR
 
+if SELINUX
+snap_confine_snap_confine_SOURCES += \
+	snap-confine/selinux-support.c \
```

Comment:
Could you ensure that selinux-support is a part of the new-style formatting (clang-format). Just look at the fmt target to see how this happens.